doc/lpqcqp.rst

```diff
@@ -109,14 +109,13 @@ to indicate equality constraints.
     coefficients, and the ``varidx`` vector contains the indices of the corresponding
     variables.
     
-.. function:: addindconstr!(m::AbstractLinearQuadraticModel, binvar, binval, varidx, coef, lb, ub)
+.. function:: addindconstr!(m::AbstractLinearQuadraticModel, binvar, binval, varidx, coef, sense, rhs)
 
     Adds a new indicator constraint to the model, of the form
 .. math::
     y = 1 \Rightarrow ax \leq b
     
-    ``binvar`` is the indicator variable (binary), and ``binval`` is the value of the binary variable. 
-    The rest of the arguments are the same as addconstr!
+    ``binvar`` is the indicator variable (binary), and ``binval`` is the value of the binary variable. Coefficients for the linear constraint are specified in a sparse format: the ``coef`` vector contains the nonzero coefficients, and the ``varidx`` vector contains the indices of the corresponding variables. ``sense`` is the sense of the linear constraint ``('<','=','>')``, and ``rhs`` is the right hand side.
```

Comment:
I played a little bit with CPLEX to get your answers

1. binvar must be the index of the variable
2. CPLEX will not complain if the variable is not marked as binary beforehand or will it mark it if declared if used in an indicator constraint. However, using the variable as indicator will make it behave as binary, even if not marked. See the examples below
3.  an indicator constraint has the form ``binvar = binval  => linearconstraint`` where ``binval`` is in {0,1}.

Let me know how do we proceed from here.

---
A few examples for 2, all of them constructed using JuMP and CPLEX: 
```
Maximize
 obj: 0 x1 + x2
Subject To
 c1: x2 <= 10
 i1: x1 = 1 -> x2 <= 5
Bounds
      x1 = 1
Binaries
 x1 
End
```
works, obj= 5

```
Maximize
 obj: 0 x1 + x2
Subject To
 c1: x2 <= 10
 i1: x1 = 1 -> x2 <= 5
Bounds
      x1 = 1
End
```
CPLEX Error  1017: Not available for mixed-integer problems. Probably caused because an LP cannot have indicators.

```
Maximize
 obj: 0 x1 + x2 + 0 x3
Subject To
 c1: x2 <= 10
 i1: x1 = 1 -> x2 <= 5
Bounds
      x1 = 1
 0 <= x3 <= 1
Binaries
 x3 
End
```
works, obj=5. Column types ``[:Cont, :Cont, :Bin]``

```
Maximize
 obj: 0 x1 + x2 + 0 x3
Subject To
 c1: x2 <= 10
 i1: x1 = 1 -> x2 <= 5
Bounds
      x1 = 0.5
 0 <= x3 <= 1
Binaries
 x3 
End
```
Infeasible. Even though x1 is not marked as binary it cannot take a value different than {0,1}